NEWS.md

```diff
@@ -64,6 +64,9 @@ Breaking changes:
   forms and second-order cone constraints. Support for these
   constraint classes depends on the solver.
 
+- The symbols `:Min` and `:Max` are no longer used as optimization senses.
```

Comment:
This might easily be misinterpreted as saying that `@objective(mode, Min, x+1)` is not supported.